tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -1123,3 +1124,356 @@ def check_extracted_metadata(self, metadata: dict):
+#         assert metadata["NWBFile"]["session_start_time"] == datetime(2014, 3, 3, 15, 21, 57, 18837, tzinfo=timezone.utc)
+
+#         # Check NWBFile metadata
+#         nwbfile_metadata = metadata["NWBFile"]
+#         assert nwbfile_metadata["session_description"] == "Session: MSession_0, MUnit: MUnit_60."
+#         assert nwbfile_metadata["experimenter"] == ["measurement"]
+#         assert nwbfile_metadata["session_id"] == "eab55dc7-173e-4fcb-8746-65274f1e5f96"
+
+#         # Check device metadata
+#         device_metadata = metadata["Ophys"]["Device"][0]
+#         assert (
+#             device_metadata["name"] == "Microscope"
+#         )  # Default metadata because this was not included in the source metadata
+#         assert device_metadata["description"] == "version: MESc 1.0, revision: 1839"
+
+#         # Check imaging plane metadata
+#         imaging_plane = metadata["Ophys"]["ImagingPlane"][0]
+#         assert imaging_plane["name"] == "ImagingPlane"
+#         assert (
+#             imaging_plane["device"] == "Microscope"
+#         )  # Default metadata because this was not included in the source metadata
+#         assert imaging_plane["imaging_rate"] == pytest.approx(31.2, rel=1e-2)
+
+#         # Grid spacing from Pixel Size X: 0.876 μm, Pixel Size Y: 0.876 μm
+#         assert imaging_plane["grid_spacing"] == [0.8757686997991967, 0.8757686997991966]
+#         assert imaging_plane["grid_spacing_unit"] == "µm"
+
+#         # Check optical channel metadata
+#         optical_channel = imaging_plane["optical_channel"][0]
+#         assert optical_channel["name"] == "UG"
+#         assert (
+#             optical_channel["description"] == "An optical channel of the microscope."
+#         )  # Default metadata because this was not included in the source metadata
+
+#         # Check two photon series metadata
+#         two_photon_series = metadata["Ophys"]["TwoPhotonSeries"][0]
+#         assert two_photon_series["name"] == "TwoPhotonSeries"
+#         assert (
+#             two_photon_series["description"] == "Imaging data from two-photon excitation microscopy."
+#         )  # Default metadata because this was not included in the source metadata
+#         assert (
+#             two_photon_series["unit"] == "n.a."
+#         )  # Default metadata because this was not included in the source metadata
+
+#         # Image dimensions from metadata: X Dimension: 512 pixels, Y Dimension: 512 pixels
+#         assert two_photon_series["dimension"] == [512, 512]
+
+#         assert (
+#             imaging_plane["description"]
+#             == "The plane or volume being imaged by the microscope. Geometric transformations: translation: [-224.19678715 -224.19678715    0.        ], rotation: [0. 0. 0. 1.], labeling_origin: [    0.       0.   -6724.23]"
+#         )
+
+
+class TestFemtonicsImagingInterfaceStaticMethods:
+    """Test static methods of FemtonicsImagingInterface."""
+
+    def test_get_available_channels_p29(self):
```

Comment:
I think this and the next tests are identical, right? I think we should remove one of them.